apps/syscfg/pkg.yml

```diff
@@ -35,7 +35,9 @@ pkg.deps:
     - "@apache-mynewt-core/sys/log/full"
     - "@apache-mynewt-core/sys/stats/full"
 #   - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
+#   - "@apache-mynewt-core/mgmt/newtmgr"
```

Comment:
Don't leave the old newtmgr here as a comment, just remove it.

Reply:
Done